crates/amaru-ledger/src/store/columns/accounts.rs

```diff
@@ -12,22 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use amaru_kernel::{cbor, Lovelace, PoolId, StakeCredential};
+use amaru_kernel::{cbor, DRep, Lovelace, PoolId, StakeCredential};
 use iter_borrow::IterBorrow;
 
 pub const EVENT_TARGET: &str = "amaru::ledger::store::accounts";
 
 /// Iterator used to browse rows from the Accounts column. Meant to be referenced using qualified imports.
 pub type Iter<'a, 'b> = IterBorrow<'a, 'b, Key, Option<Row>>;
 
-pub type Value = (Option<PoolId>, Option<Lovelace>, Lovelace);
+pub type Value = (
+    Option<(Option<PoolId>, Option<DRep>)>,
```

Comment:
Required by DiffBind feature to bind a previous value.

Reply:
Wat

Reply:
I improved it a bit just for you